SQF/dayz_code/stringtable.xml

```diff
@@ -15870,12 +15874,16 @@
 			<English>Trade successfull.</English>
 			<Russian>Продажа завершена.</Russian>
 		</Key>
-		<Key ID="STR_EPOCH_TRADE_SUCCESS">
-			<English>Trade successfull, payed %1 worth of items.</English>
+		<Key ID="STR_EPOCH_TRADE_BUY_SUCCESS">
+			<English>Trade successfull, paid:%1.</English>
 			<Russian>Продано! С вас %1.</Russian>
 		</Key>
+		<Key ID="STR_EPOCH_TRADE_SELL_SUCCESS">
+			<English>Trade successfull, received:%1.</English>
```

Comment:
This is a duplicate of "STR_EPOCH_TRADE_SUCCESS_CHANGE", so you can remove that one now that you renamed it. Also the Russian translation will be the same as "STR_EPOCH_TRADE_SUCCESS_CHANGE" minus the %2.